intelmq/bots/outputs/xmpp/output.py

```diff
@@ -1,4 +1,24 @@
-# -*- coding: utf-8 -*-
+"""
+XMPP Output Bot
+Connects to a XMPP Server and sends data to a user.
+
+TODO: Introduce Multi User Chat like in XMPP Collector
+
+Requires Python >= 3.4
+Requires sleekxmpp >= 1.0.0-beta5
```

Comment:
> 1.0.0-beta5 does not exist on pypi.
But on Ubuntu 14.04 Repositories.

>  1.0beta6.1 exists, but does not have IqError and IqTimeout
That's funny because https://github.com/fritzy/SleekXMPP/blob/sleek-1.0-Beta6.1/sleekxmpp/exceptions.py has these classes!There seems to be a problem with pip?